rs/registry/canister/src/common/test_helpers.rs

```diff
@@ -96,6 +96,7 @@ pub fn get_invariant_compliant_subnet_record(node_ids: Vec<NodeId>) -> SubnetRec
 pub fn prepare_registry_with_nodes(
     start_mutation_id: u8,
     nodes: u64,
+    node_operator_id: Option<PrincipalId>,
```

Comment:
nit: maybe this should take a `PrincipalId`, and the caller can be responsible for passing the default one if they don't have anything better to pass? cc @max-dfinity